country.rb

```diff
@@ -0,0 +1,48 @@
+require_relative 'country_module'
+class Country
+
+  include Check_country
+
+  def initialize(country_name ,population, gross_domestic_production, states, army_strength, state_of_country)
+    @country_name = country_name
+    @population = population
+    @gross_domestic_production = gross_domestic_production
+    @states = states
```

Comment:
It's better to use clear and consistent names in method parameters. It could be more meaningful, such as state_count.